README.md

```diff
@@ -110,6 +110,18 @@ You will need git-for-windows, Visual Studio 2015 or newer and CMake 3.13 or new
 
     For seeing some output from the samples, you will obviously have to always execute both the sender snd udpcap_reciever sample.
 
+## CMake Options
+
+You can set the following CMake Options to control how Udpcap is supposed to build:
+
+**Option**                                     | **Type** | **Default** | **Explanation**                                                                                                 |
+|----------------------------------------------|----------|-------------|-----------------------------------------------------------------------------------------------------------------|
+| `UDPCAP_BUILD_SAMPLES`                       | `BOOL`   | `ON`        | Build the Udpcap (and asio) samples for sending and receiving dummy data                                        |
+| `UDPCAP_THIRDPARTY_ENABLED`                  | `BOOL`   | `ON`        | Activate / Deactivate the usage of integrated dependencies.                                                     |
+| `UDPCAP_THIRDPARTY_USE_BUILTIN_NPCAP`        | `BOOL`   | `ON`        | Fetch and build against an integrated Version of the npcap SDK. <br>Only available if `UDPCAP_THIRDPARTY_ENABLED=ON` |
+| `UDPCAP_THIRDPARTY_USE_BUILTIN_PCAPPLUSPLUS` | `BOOL`   | `ON`        | Fetch and build against an integrated Version of Pcap++. <br>_Only available if `UDPCAP_THIRDPARTY_ENABLED=ON`_        |
+| `UDPCAP_THIRDPARTY_USE_BUILTIN_ASIO`         | `BOOL`   | `ON`        | Fetch and build against an integrated Version of asio. <br>Only available if `UDPCAP_THIRDPARTY_ENABLED=ON`          |
+| `CMAKE_BUILD_SHARED_LIBS`                    | `BOOL`   |             | Not a udpcap option, but use this to control whether you want to have a static or shared library                |
```

Comment:
@FlorianReimold The default CMake option for this is `BUILD_SHARED_LIBS` https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html. Please rather use the standard variable 😉
(Also official Conan generators will generate this variable when declaring a static / shared option. )

Reply:
This was just a typo in the readme. I am using the standard `BUILD_SHARED_LIBS` variable (or rather CMake uses it and I didn't interfere with that)